clueless/state.py

```diff
@@ -15,14 +15,24 @@
 from clueless.error import ApiError
 from enum import IntEnum, auto
 
+# Ask for clarification on wording
```

Comment:
Not currently used, and probably won't need all of these. I think we'd just need something for the different phases of the player's turn.

Reply:
I'm sure you saw this already, but I started using it in the register instance. Is this something we'd want to change? I'll keep it the same for now.